src/adapter/src/catalog/state.rs

```diff
@@ -220,7 +220,7 @@ impl CatalogState {
             state: Cow::Borrowed(self),
             unresolvable_ids: BTreeSet::new(),
             conn_id: SYSTEM_CONN_ID.clone(),
-            cluster: "default".into(),
+            cluster: "quickstart".into(),
```

Comment:
Can we use `mz_catalog::durable::initialize::DEFAULT_USER_CLUSTER_NAME` here?

Reply:
This made me realize we shouldn't use a static value for a session-less user, and instead we should use whatever value is in the system vars, updated!